cmd/porter/mixins_test.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/deislabs/porter/pkg/porter"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBuildListMixinsCommand_DefaultFormat(t *testing.T) {
```

Comment:
nit: should this actually be named the `*_AlternateFormat` test?  (non-default `-o json` passed)

Reply:
D'oh! I think I got interrrupted mid-test and these 2 aren't named right and one isn't finished! 😊 Lemme fix.

Reply:
OK, all fixed in the latest commit. Thanks for noticing!